tools/csv-merger/csv-merger.go

```diff
@@ -333,6 +333,13 @@ func main() {
 
 			strategySpec := csvStruct.Spec.InstallStrategy.StrategySpec
 
+			// temporary workarounf for RHBZ#....
```

Comment:
Is there actually a bugzilla on this? If so, it's worthwhile to link it so a future maintainer will have the context for why it was added (and when can be removed).